components/studio/libexec/hab-studio-type-default.sh

```diff
@@ -84,6 +86,12 @@ exec $HAB_ROOT_PATH/bin/hab pkg exec core/hab-plan-build hab-plan-build \$*
 EOF
   $bb chmod $v 755 $HAB_STUDIO_ROOT$HAB_ROOT_PATH/bin/build
 
+  $bb cat <<TAIL_SUP > $HAB_STUDIO_ROOT$HAB_ROOT_PATH/bin/tail_sup
```

Comment:
To me `log<s>` feels vague. Its unclear if it is build log or supervisor logs. In a studio, my assumption would be build logs.